packages/react/src/global-nav-profile-menu/GlobalNavProfileMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import type { ReactNode } from "react";
+
+import * as RadixCollapsible from "@radix-ui/react-collapsible";
+
+import { Box, type BoxProps } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import { IconEllipsis } from "../icons/IconEllipsis";
+import { Text } from "../text";
+import * as styles from "./GlobalNavProfileMenu.css";
+
+export type GlobalNavProfileMenuProps = BoxProps<
+  "div",
+  {
+    avatar: ReactNode;
+    name?: string;
+    organization?: string;
+  }
+>;
+
+export const GlobalNavProfileMenu = ({
+  avatar,
+  name,
+  organization,
+}: GlobalNavProfileMenuProps) => {
+  return (
+    <Flex {...styles.wrapper({})}>
+      <Box asChild {...styles.picture()}>
+        {avatar}
+      </Box>
```

Comment:
Can we reuse the avatar component here?

Reply:
I am afraid we can't. The borderRadius doesn't match here